jinja2/includes/auth-modal.html

```diff
@@ -0,0 +1,19 @@
+<div id="auth-modal" class="modal hidden">
+  <section class="auth-providers" tabindex="-1" role="dialog" aria-modal="true" aria-labelledby="auth-modal-main-heading">
+    <header>
+      <h2 id="auth-modal-main-heading">{{ _('Sign In') }}</h2>
+    </header>
+    <p>{{ _('Sign in to enjoy the benefits of an MDN account. If you haven’t already created an MDN profile, you will be prompted to do so after signing in.') }}</p>
+    <div class="auth-button-container">
+      <a href="{{ provider_login_url('github') }}" class="github-auth" data-first-focusable="true">
+        {{ _('Sign in with Github') }}
+      </a>
+      <a href="{{ provider_login_url('github') }}" class="google-auth">
+        {{ _('Sign in with Google') }}
+      </a>
+    </div>
+    <button id="close-modal" class="close-modal" data-last-focusable="true">
+      <span>{{ _('Close modal') }}</span>
+    </button>
+  </section>
+</div>
```

Comment:
Thinking over all of this some more, I am thinking that this can be made into a base template that you can then extend when creating a modal. The base template might look something like this:

```
<div id="{{ modal_id }}" class="modal hidden">
  <section class="{{ modal_content_class }}" tabindex="-1" role="dialog" aria-modal="true" aria-labelledby="modal-main-heading">
    <header>
      <h2 id="modal-main-heading">{{ modal_heading }}</h2>
	</header>
    {% block modal_content %}{% endblock %}
    <button id="close-modal" class="close-modal" data-last-focusable="true">
      <span>{{ _('Close modal') }}</span>
    </button>
  </section>
</div>
```

Something like that. Thoughts?

Reply:
So, I went ahead and did what I talked about above :) Let me know if this was not a good move ;p